test/games/strategy/engine/data/TestDelegateBridge.java

```diff
@@ -30,7 +31,7 @@
   private PlayerID m_id;
   private String m_stepName = "no name specified";
   private IDisplay m_dummyDisplay;
-  private final DummySoundChannel m_soundChannel = new DummySoundChannel();
+  private final ISound m_soundChannel = mock(ISound.class);
```

Comment:
I know, but as experienced before, the Test delegate is used very often. And is used deep in the test logic... Will do that in some of the next PRs